### PR TITLE
Adding Avatar

### PR DESCRIPTION
**Pull request by chahatbansal**
* Added Avatar

* Updated index.html

Changes Preview : https://chahatbansal.github.io/gci17.fossasia.org/
screen shot 2017-11-30 at 1 36 26 pm

### CODE REVIEW

**Comment by chahatbansal** [comment]
@stealthanthrax

**Review by sansyrox — APPROVED** [review]
@chahatbansal ,add a screenshot of your work regarding any UI/UX change.
Rest LGTM 🚀

**Comment by chahatbansal** [comment]
@mariobehling Please merge this PR.

File: index.html [hunk 1]
```diff
@@ -287,7 +287,7 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        <div class="container">
 		            <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img alt="Pic" src="img/avatar1.png">
+		                    <img class="col-2" alt="Chahat Bansal" src="img/chahat.png">
```
Comment (SaptakS) [inline index.html:289]: I still don't think this extra class is needed. Can you please explain why you are adding this?

**Comment by chahatbansal** [comment]
@SaptakS  One mentor already approved it and I made changes according to what he said. I'm not able to understand to whom I should follow. The mentorship system in FOSSASIA is really bad, I'm trying to do a simple task from last 12 hours, and I'm told to do this and that. Everything should be clearly mentioned on the task page with steps on how to do it. I checked the previous PR made, and it didn't required any kind of class to be added. I made a PR without class, but it wasn't accepted and I was told to add class to it, and now you are asking me the reason behind it. And why I need to add a screenshot, when it's not mentioned on the task page. 
Previous PR's : #40 #41

**Review by ParthS007 — APPROVED** [review]
LGTM ! :+1:

**Comment by ParthS007** [comment]
@chahatbansal Screenshot is used for reviewing the changes made in PR, that makes review process fast . Please post the PR link to the gitter channel tagging mario behling and ask him to review your work, that is what you have to do after every PR you made. I think it is all clear now. :+1:

**Review by sansyrox — APPROVED** [review]

**Comment by rishabh-bansal** [comment]
@stealthanthrax  @ParthS007  Thank you for guiding me :smile:

**Comment by chahatbansal** [comment]
@stealthanthrax @ParthS007  thankyou for guiding me 😄

**Pull request merged**